### DISCUSSION TITLE
请问可以离线加载吗，或者引入自己的离线服务。

### DISCUSSION THREAD

**Discussion by HHANG52** [zouyaoji/vue-cesium#450] [Q&A]
esiumPath ： --- 'https://unpkg.com/cesium@latest/Build/Cesium/Cesium.js'这里可以离线使用，地图怎么引入自己的离线图层呢 ，用哪个组件，还是用原生的呢。

**Comment by zouyaoji** [comment]
可以的哈，你自己发布或者放前端工程都可以。Cesium官方，第三方基于Cesium的都行。
